src/app/api/stories/route.ts

```diff
@@ -81,7 +81,7 @@ export async function GET(req: NextRequest) {
               ],
             }
           : {}),
-        staffPick: staff_pick,
```

Comment:
this should probably be replaced with something else checking staff pick status instead of deleted

https://www.prisma.io/docs/orm/prisma-client/queries/relation-queries#filter-on-absence-of--to-one-relations
https://stackoverflow.com/questions/69184087/how-do-i-filter-if-an-optional-relationship-exists

maybe something here helps?

Reply:
I think I can replace with this if the user want to get story that have staff pick in the search param
`staffPick: { isNot: null }`